MapboxNavigation/CarPlayManager.swift

```diff
@@ -57,7 +57,6 @@ public class CarPlayManager: NSObject {
 
     public fileprivate(set) var mainMapTemplate: CPMapTemplate?
     public fileprivate(set) weak var currentNavigator: CarPlayNavigationViewController?
-    public static let CarPlayWaypointKey: String = "MBCarPlayWaypoint"
```

Comment:
I would move this key into `AppDelegate+CarPlay.swift`. There are a few references to this string in that file.